frontend/packages/component-library/src/actionBlock/ActionBlock.tsx

```diff
@@ -0,0 +1,160 @@
+import classNames from 'classnames';
+import {HTMLAttributes} from 'react';
+
+import {LinkButton, LinkButtonProps} from '@/button';
+import {
+  Heading3,
+  BodyThreeText,
+  OverlineTwoText,
+  StrongText,
+} from '@/typography';
+
+import moduleStyles from './actionBlock.module.scss';
+
+export interface ActionBlockProps extends HTMLAttributes<HTMLDivElement> {
+  /** Action Block title */
+  title?: string;
+  /** Action Block description */
+  description?: string;
+  /** Action Block image */
+  image?: string;
+  /** Action Block overline */
+  overline?: string;
+  /** Action Block Details */
+  details?: {
+    /** Detail label */
+    label: string;
+    /** Detail text */
+    description: string;
+  };
+  /** Primary button props */
+  primaryButton?: LinkButtonProps;
+  /** Secondary button props */
+  secondaryButton?: LinkButtonProps;
+  /** Action Block background */
+  background?: 'primary' | 'secondary';
+  /** Action Block custom className */
+  className?: string;
+}
+
+export const getImage = (image?: string) => {
+  if (!image) return null;
+  return (
+    <figure>
+      <img src={image} alt="" loading={'lazy'} />
```

Comment:
Should the alt text be the detail label/description?

Reply:
We don't want alt text here since these are decorative images.